docs/node/workflows.md

```diff
@@ -57,62 +57,83 @@ It can be used in any NodeJS process e.g an express app and does not depend on t
-and [`withTimeout`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#withtimeout).
+Scopes all return native JS Promises, so you can use all the familiar Promise APIs like `Promise.all` and `Promise.race` to model your async logic.
+Other APIs you can use:
 
-When a `CancellationScope` is cancelled, it propagates cancellation in any child scopes and of any cancellable operations created within it, such as:
+- `CancellationScope.current()`: get the current scope
+- `scope.cancel()`: cancel all operations inside a `scope`
+- `scope.run()`: run an async function within a `scope`
+- `scope.cancelRequested`: a promise that resolves when a scope cancellation is requested, e.g when Workflow code calls `cancel()` or the entire Workflow is cancelled by an external client.
+
+When a `CancellationScope` is cancelled, it propagates cancellation to any child scopes and any cancellable operations created within it, such as:
 
 - Activities
 - Timers (created with the [`sleep`](https://nodejs.temporal.io/api/modules/workflow#sleep) function)
 - [`Trigger`](https://nodejs.temporal.io/api/classes/workflow.trigger)s
 
-## Examples
-
-### Cancel a timer from Workflow code
+### Internal Cancellation example
 
 <!--SNIPSTART nodejs-cancel-a-timer-from-workflow-->
 <!--SNIPEND-->
 
-### Alternatively, the preceding can be written as
+Alternatively, the preceding can be written as:
 
 <!--SNIPSTART nodejs-cancel-a-timer-from-workflow-alternative-impl-->
 <!--SNIPEND-->
 
-### Run multiple activities with a single deadline
+### External Cancellation example
 
-<!--SNIPSTART nodejs-multiple-activities-single-timeout-workflow-->
+Handle Workflow cancellation by an external client while an Activity is running:
+
+<!--SNIPSTART nodejs-handle-external-workflow-cancellation-while-activity-running-->
 <!--SNIPEND-->
 
-### `nonCancellable` prevents cancellation from propagating to children
+### `nonCancellable` example
+
+`CancellationScope.nonCancellable` prevents cancellation from propagating to children:
 
 <!--SNIPSTART nodejs-non-cancellable-shields-children-->
 <!--SNIPEND-->
 
-### `cancelRequested` may be awaited upon to make Workflow aware of cancellation while waiting on `nonCancellable` scopes
+### `withTimeout` example
+
+A very common operation is to cancel one or more activities if a deadline elapses, `withTimeout` creates a `CancellationScope` that is automatically cancelled after a given timeout.
+
+<!--SNIPSTART nodejs-multiple-activities-single-timeout-workflow-->
+<!--SNIPEND-->
+
+### `scope.cancelRequested`
+
+You can await `cancelRequested` to make Workflow aware of cancellation while waiting on `nonCancellable` scopes:
 
 <!--SNIPSTART nodejs-cancel-requested-with-non-cancellable-->
 <!--SNIPEND-->
 
-### Handle Workflow cancellation by an external client while an Activity is running
+Callbacks are not particularly useful in Workflows because all meaningful asynchronous operations return Promises.
```

Comment:
```suggestion
### CancellationScopes and callbacks 

Callbacks are not particularly useful in Workflows because all meaningful asynchronous operations return Promises.
```